backend/src/index.ts

```diff
@@ -0,0 +1,7 @@
+import { createServer } from "./server";
+
+const server = createServer();
+
+server.listen().then(({ url }) => {
+    console.log(`🚀  Server ready at ${url}`);
+});
```

Comment:
you could just: import Server from './server.js'; (no createServer needed we guess)
the rest is fine... const server and listen --- all good here